packages/jest-emotion/README.md

```diff
@@ -16,9 +16,18 @@ The easiest way to test React components with emotion is with the snapshot seria
 // jest.config.js
 module.exports = {
   // ... other config
-  snapshotSerializers: [
-    'jest-emotion' /* if needed other snapshotSerializers should go here */
-  ]
+  snapshotSerializers: ['jest-emotion']
+}
+```
+
+To assist with shallow rendering, there's a custon enzyme snapshot serializer, that includes the `enzyme-to-json`
+serializer, which is available by importing `jest-emotion/enzyme`:
```

Comment:
Could you add a note mentioning that people should remove the enzyme-to-json if they have it?

Reply:
sounds good